editor/icons/source/icon_GUI_tree_arrow_right.svg

```diff
@@ -71,7 +71,7 @@
      id="layer1"
      transform="translate(0,-1040.3622)">
     <path
-       style="fill:none;stroke:#ffffff;stroke-width:2;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:0.39215687"
```

Comment:
@djrm it actually looks pretty nice with a slightly darker color too. it's only a tiny bit darker. I don't know what issues I will get when adding ffffff -> 000000.
we maybe could do fefefe. so all colors which should become dark use fefefe and all colors which should stay white use ffffff